mitodl/open-edx-plugins
# build: update the Django versions support(3.2)

## Related Ticket:
#14 , #15 

## What this PR does:
- Tightens the boundary for the Django version to be used (`above 2.0 & below 4.0`) 
- Updates the poetry config

## How to test
- Generate new plugin builds for `ol-openedx-logging` & `ol-openedx-sentry` and install them into the Maple release of edX and make sure that things work fine.
